### ISSUE TITLE
Outdated Readme

### ISSUE THREAD

**Issue by codingyourlife** [goooseman/storybook-addon-i18n#14]
I was just looking into this addon and the readme really confused me because I didn't know the mentioned files `config.js` and `addons.js.`.

After a bit of research the files have been renamed in the storybook project as you can read about here: [https://github.com/storybookjs/storybook/blob/v5.3.18/MIGRATION.md#from-version-52x-to-53x](https://github.com/storybookjs/storybook/blob/v5.3.18/MIGRATION.md#from-version-52x-to-53x)

Because this change was in version 5.3 and the project is now v6.0 I have doubts that this plugin even works.

So basically my question is: Does this plugin still work?

And my request is: Please update the Readme with the current filenames.

Also the story's config part isn't so clear because the storiesOf() function is not part of the getting started guide in storybook.

**Comment by goooseman** [comment]
This repository is no longer maintained, but I've updated the filenames in the README (https://github.com/goooseman/storybook-addon-i18n/commit/ea9b66ee94becd6a78d6d012b18cae0505f1f2ca)

Please, pay attention to a new deprecation section of the README, which specifies how this functionality can be achieved without plugins
